moon/controller_round3.py

```diff
@@ -111,6 +113,7 @@ def __init__(self, config, bus):
         self.basemarker_whole_scan_history = []
         self.basemarker_radius = None
         self.centering = False
+        self.going_around_count = 0
```

Comment:
why you do not use time here? It is how many iterations are you going around, right?

Reply:
Imagine you're trying to go around a homebase looking for the triangle. Lets say you run into an issue that prevents you from continuing (e.g., terrain starts to be hilly and the robot keeps skidding and can't maintain the circular trajectory). Maybe you hit an exception (pitch protection), you lose track of the homebase so the robot pulls back, tries to locate homebase again, go back to it and start going around.

What this does is that next time it tries to go around, it will go around in the opposite direction, hopefully avoiding the area that caused it trouble the first time. 

Of course, if the objectionable terrain is in front of the triangle, you're out of luck either way.